src/Delegate/ArraySerializable.php

```diff
@@ -34,6 +34,10 @@ public function getArrayCopy()
 					// If 'getSomething' doesn't exist, try to use 'isSomething'
 					$getter = 'is' . ucfirst( $property );
 				}
+				// Finally simply use the property
+				if( !method_exists($this, $getter) ) {
+					$getter = $property;
+				}
```

Comment:
I see the tests are running fine when I comment out this addition, so I will update the tests to try and make sure it fails in the case you described in #52.